library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopFederatedIT.java

```diff
@@ -274,4 +277,111 @@ void shouldTraverseEdgesFromVertexAndReturnNames() {
                 .containsExactly(EXPECTED_PERSON_2_VERTEX_MAP.get("name"),
                         EXPECTED_SOFTWARE_1_VERTEX_MAP.get("name"));
     }
+
+    @Test
+    void shouldReturnVerticesByLabelFromSpecificGraph() {
+        // Given
+        GraphTraversalSource g = gafferPopGraph.traversal();
+        final List<String> graphOptions = Arrays.asList("gaffer.federatedstore.operation.graphIds:graphA");
+
+        // When
+        List<Map<Object, Object>> result = g.with(GafferPopGraphVariables.OP_OPTIONS, graphOptions).V()
+                .hasLabel(PERSON_GROUP)
+                .elementMap().toList();
+
+        // Then
+        assertThat(result)
+                .containsExactlyInAnyOrder(EXPECTED_PERSON_1_VERTEX_MAP, EXPECTED_PERSON_2_VERTEX_MAP,
+                        EXPECTED_PERSON_3_VERTEX_MAP);
+    }
+
+    @Test
+    void shouldReturnFilteredVerticesFromSpecificGraph() {
+        // Given
+        GraphTraversalSource g = gafferPopGraph.traversal();
+        final List<String> graphOptions = Arrays.asList("gaffer.federatedstore.operation.graphIds:graphA");
+
+        // When
+        List<Object> result = g.with(GafferPopGraphVariables.OP_OPTIONS, graphOptions).V()
+                .outE(CREATED_EDGE_GROUP).has(WEIGHT_PROPERTY, P.gt(0.4))
+                .values(WEIGHT_PROPERTY)
+                .toList();
+
+        // Then
+        assertThat(result)
+                .containsExactly(1.0);
```

Comment:
Is this correct? There isn't a created edge with weight of `1.0` in your test graph, I think this is related to issue #3203

Reply:
ah you are right, I shall fix this. Thanks :)

Reply:
As this is a problem in an earlier test that has also been missed, I have temporarily disabled both tests until #3205 is merged in.